futurecollars/src/main/java/com/futurecollars/lekcja3/zadanie4/TimeConverter.java

```diff
@@ -0,0 +1,9 @@
+package com.futurecollars.lekcja3.zadanie4;
+
+public class TimeConverter {
+
+    public static int convertingMinutesToSeconds(int Minutes) {
```

Comment:
nazwa argumentu także powinna być z małej litery, jest to zwykła zmienna w środku metody